src/android/adb.ts

```diff
@@ -18,14 +19,24 @@ export function getAdbPath(customPath?: string): string {
     return path.resolve(customPath);
   }
 
-  return process.env.ANDROID_HOME
-    ? `${process.env.ANDROID_HOME}/platform-tools/adb`
-    : 'adb';
+  if (isWin) {
+    return process.env.ANDROID_HOME
+      ? `${process.env.ANDROID_HOME}\\platform-tools\\adb`
+      : 'adb';
+  } else {
+    return process.env.ANDROID_HOME
+      ? `${process.env.ANDROID_HOME}/platform-tools/adb`
+      : 'adb';
+  }
 }
 
 export function spawnLogcatProcess(adbPath: string): ChildProcess {
   try {
-    execSync(`'${adbPath}' logcat -c`);
+    if (isWin) {
+      execSync(`${adbPath} logcat -c`);
```

Comment:
Using `execSync(`${adbPath} logcat -c`)` should be fine in Unix/Linux as well, so we might be able to get rid of this `if` statement.